libindy/src/services/metrics/mod.rs

```diff
@@ -125,10 +150,8 @@ mod test {
 
         metrics_service.append_command_metrics(&mut metrics_map);
 
-        assert_eq!(metrics_map.len(), COMMANDS_COUNT * 4);
-        assert!(metrics_map.contains_key("issuer_command_create_schema_queued_commands_count"));
-        assert!(metrics_map.contains_key("issuer_command_create_schema_queued_commands_duration_ms"));
-        assert!(metrics_map.contains_key("issuer_command_create_schema_executed_commands_count"));
-        assert!(metrics_map.contains_key("issuer_command_create_schema_executed_commands_duration_ms"));
+        assert!(metrics_map.contains_key("commands_count"));
+        assert!(metrics_map.contains_key("commands_duration"));
```

Comment:
```suggestion
        assert!(metrics_map.contains_key("commands_duration_ms"));
```
It is a good practice to indicate unit of measure in metrics name, let's stick to it.